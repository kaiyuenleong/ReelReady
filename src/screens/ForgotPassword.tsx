import React, { Component } from "react";
import { View, Text } from "react-native";

class ForgotPassword extends Component {
	render() {
		return (
			<View>
				<Text>Forgot Password</Text>
			</View>
		)
	}
}

export default ForgotPassword;