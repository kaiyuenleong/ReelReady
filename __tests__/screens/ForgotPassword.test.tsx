import React from "react";
import { render } from "react-native-testing-library";

import ForgotPassword from "../../src/screens/ForgotPassword";

describe('<ForgotPassword />', () => {
  it('renders correctly', () => {
    const tree = render(<ForgotPassword />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})