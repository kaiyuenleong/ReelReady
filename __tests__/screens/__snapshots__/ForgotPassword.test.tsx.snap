// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForgotPassword /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ViewManagerAdapter_ExpoLinearGradient
    proxiedProperties={
      Object {
        "colors": Array [
          4289198296,
          4283231449,
          4279833944,
          4279304248,
        ],
        "endPoint": undefined,
        "locations": Array [
          0,
          0.1,
          0.8,
          1,
        ],
        "startPoint": undefined,
      }
    }
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text>
      Forgot Password
    </Text>
  </ViewManagerAdapter_ExpoLinearGradient>
</View>
`;
